pkg/manager/member/tidb_member_manager.go

```diff
@@ -47,6 +47,10 @@ const (
 	serverCertPath = "/var/lib/tidb-server-tls"
 	// serviceAccountCAPath is where is CABundle of serviceaccount locates
 	serviceAccountCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	// tlsSecretRootCAKey is the key used in tls secret for the root CA.
+	// When user use self-signed certificates, the root CA must be provided. We
+	// following the same convention used in Kubernetes service token.
+	tlsSecretRootCAKey = corev1.ServiceAccountRootCAKey
```

Comment:
tls secret created by cert-manager use the same key `ca.crt` too. https://github.com/jetstack/cert-manager/blob/release-0.13/pkg/apis/meta/v1/types.go#L63-L65